SwiftBasicsExercise.playground/Pages/Dictionary_6.2.1.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,59 @@
+//MARK: Dictionary excercise 1
+var code = [ "a" : "b",
+             "b" : "c",
+             "c" : "d",
+             "d" : "e",
+             "e" : "f",
+             "f" : "g",
+             "g" : "h",
+             "h" : "i",
+             "i" : "j",
+             "j" : "k",
+             "k" : "l",
+             "l" : "m",
+             "m" : "n",
+             "n" : "o",
+             "o" : "p",
+             "p" : "q",
+             "q" : "r",
+             "r" : "s",
+             "s" : "t",
+             "t" : "u",
+             "u" : "v",
+             "v" : "w",
+             "w" : "x",
+             "x" : "y",
+             "y" : "z",
+             "z" : "a"]
+//
+let inputStr = "uijt nfttbhf jt ibse up sfbe"
+var arrKey: [String] = []
+var arrVal: [String] = []
+var decode: [String: String] = [:]
+
+code.forEach { (key, val) in
+    arrKey.append(key)
+    arrVal.append(val)
+}
+
```

Comment:
We don't need the arrKey arrVal,
directly inverse the dictionary to get decoding dictionary